dallinger/command_line.py

```diff
@@ -501,25 +501,25 @@ def expire(app, sandbox):
                 'If this experiment was run in the MTurk sandbox, use: '
                 '`dallinger expire --sandbox --app {}`'.format(app)
             )
-    if not success:
+    if exit and not success:
```

Comment:
I'm sure this wouldn't be here if you didn't need it, but why the explicit sys.exit()?

Reply:
It ensures a failure to find/expire HITs is treated as a UNIX command error (non-zero exit code). In the case of the destroy command we don't want to exit before actually destroying the Heroku app, so we don't treat that as a command error (though a failure to destroy the Heroku app would be one).